<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>603</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>603</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NetWork</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/prefix1/icon/title.png</normaloff>:/new/prefix1/icon/title.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>830</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1" columnstretch="1,3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>当前连接</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>分组及收藏列表</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTreeWidget" name="treeWidget">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>客户端模式</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/icon/clientmodel.png</normaloff>:/new/prefix1/icon/clientmodel.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>服务器模式</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/icon/servermodel.png</normaloff>:/new/prefix1/icon/servermodel.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>串口模式</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/new/prefix1/icon/chuankoumodel.png</normaloff>:/new/prefix1/icon/chuankoumodel.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QTreeWidget" name="treeWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="autoScrollMargin">
          <number>16</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(237, 237, 237);</string>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>操作(O)</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置(S)</string>
    </property>
    <widget class="QMenu" name="menu_language">
     <property name="title">
      <string>语言/language</string>
     </property>
     <addaction name="action_11"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menu_language"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(V)</string>
    </property>
    <addaction name="action_10"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>模板(M)</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
   </widget>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_S"/>
   <addaction name="menu_H"/>
   <addaction name="menu_M"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="action_6"/>
   <addaction name="action_5"/>
   <addaction name="action_7"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/createconnect.png</normaloff>:/new/prefix1/icon/createconnect.png</iconset>
   </property>
   <property name="text">
    <string>创建连接</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/createserver.png</normaloff>:/new/prefix1/icon/createserver.png</iconset>
   </property>
   <property name="text">
    <string>创建服务器</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/createchuankou.png</normaloff>:/new/prefix1/icon/createchuankou.png</iconset>
   </property>
   <property name="text">
    <string>创建串口</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/allconnect.png</normaloff>:/new/prefix1/icon/allconnect.png</iconset>
   </property>
   <property name="text">
    <string>全部连接</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/sloveall.png</normaloff>:/new/prefix1/icon/sloveall.png</iconset>
   </property>
   <property name="text">
    <string>收藏全部</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/allunconnect.png</normaloff>:/new/prefix1/icon/allunconnect.png</iconset>
   </property>
   <property name="text">
    <string>全部断开</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/exit.png</normaloff>:/new/prefix1/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/delete.png</normaloff>:/new/prefix1/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除全部客户端</string>
   </property>
  </action>
  <action name="action_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/icon/delete.png</normaloff>:/new/prefix1/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除全部服务器</string>
   </property>
  </action>
  <action name="action_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="action_13">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>属性栏</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>模板设置</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
